newrelic/hooks/framework_django_py3.py

```diff
@@ -19,8 +19,8 @@ async def _nr_wrapper_BaseHandler_get_response_async_(
     request = _bind_get_response(*args, **kwargs)
 
     if hasattr(request, '_nr_exc_info'):
-        if not ignore_status_code(response.status_code):
-            record_exception(*request._nr_exc_info)
+        if not should_ignore_error(request._nr_exc_info, response.status_code):
+            notice_error(request._nr_exc_info)
```

Comment:
Same as the other Django file. The if statement is unnecessary, you can pass response.status_code into notice_error and let it handle evaluating.